contracts/finality/src/contract.rs

```diff
@@ -55,6 +57,18 @@ pub fn query(
         QueryMsg::LastPubRandCommit { btc_pk_hex } => Ok(to_json_binary(
             &get_last_pub_rand_commit(deps.storage, &hex::decode(&btc_pk_hex)?)?,
         )?),
+        QueryMsg::ListPubRandCommit {
+            btc_pk_hex,
+            start_after,
+            limit,
+            reverse,
+        } => Ok(to_json_binary(&get_pub_rand_commit(
+            deps.storage,
+            &hex::decode(&btc_pk_hex)?,
+            start_after,
+            limit,
+            reverse,
+        )?)?),
```

Comment:
```suggestion
        } => Ok(to_json_binary(&list_pub_rand_commit(
            deps.storage,
            &hex::decode(&btc_pk_hex)?,
            start_after,
            limit,
            reverse,
        )?)?),
```

as this function is not really a getter